drools-benchmarks-parent/drools-benchmarks-module/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>drools-benchmarks-parent</artifactId>
+    <groupId>org.drools</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>kjar</packaging>
+
+  <artifactId>drools-benchmarks-module</artifactId>
+  <name>Drools 8 Benchmarks Module</name>
+  <description>Drools module to use for benchmarking purpose</description>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-benchmarks-common</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-benchmarks-common</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
```

Comment:
The PR check build failures are caused by this. Please remove, it is not necessary as drools-bom is imported in the drools-benchmarks-parent module.